keymaps/fix-clashing-shortcuts.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -euo pipefail
+
+# Disables Ubuntu shortcuts that clash with IntelliJ Idea (and probably other
+# Jetbrain products).
+#
+# Creates a backup file to restore the previous settings. To not have some
+# shortcuts disabled, comment them out in the `KEYS` array.
+#
+# Tested on : Ubuntu 15.10
```

Comment:
@MichaelOrlov was this tested on Ubuntu 18.04 too?

Reply:
Yes. I tested it on my machine